README.md

```diff
@@ -977,8 +977,8 @@ if err == nil {
     }
 }
 
-// Update will override all fields as is. Use carefully.
-res, err := descopeClient.Management.AccessKey().Update(context.Background(), "access-key-id", "updated-name")
+// If permittedIPs is nil, then the existing list of permitted IPs will be preserved.
```

Comment:
What's the way to clear them? empty array? If so, let's metion this as well.

Reply:
good point, added 👍

Reply:
I don't think it's idiomatic Go to have different behavior between an empty slice and a `nil` slice value.

Code like this will not behave as expected. It'll update the permitted IPs if `addBackendIP` is `true` but will keep the same value if `addBackendIP` is `false`. 

```go
	addBackendIP := true

	var permittedIPs []string
	if addBackendIP {
		permittedIPs = append(permittedIPs, "1.2.3.4")
	}

	println(permittedIPs == nil)
```